java/arcs/core/analysis/AccessPathLabels.kt

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.analysis
+
+import arcs.core.data.AccessPath
+
+/**
+ * An abstract value that keeps track of [InformationFlowLabels] for different [AccessPath] values.
```

Comment:
Nit: I'd suggest using `[AbstractValue]` instead of `abstract value` just to make it clear you're talking about a specific type rather than the normal understanding of the phrase.